src/reactapp/src/components/FetchHelpers.js

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+import { transform as _transform } from "lodash-es";
+
+// TODO: replace hard-coded with some sort of runtime setting
+const BaseUrl = "https://localhost:7065";
+
+let token = "";
+
+const modify = async (verb, route, values) => {
+  if (!route.startsWith("/")) {
+    route = `/${route}`;
+  }
+
+  const request = {
+    method: verb,
+    headers: { "Content-Type": "application/json" },
+    body: JSON.stringify(values),
+  };
+
+  if (token) {
+    request.headers["Authorization"] = `bearer ${token}`;
+  }
+
+  const url = `${BaseUrl}${route}`;
+
+  return await fetch(url, request);
+
+  // TODO: might be good to detect token failure here, maybe redirecting to
+  // signin. Or at least insuring there is a user-friendly error message.
+};
+
+const post = async (route, values) => {
+  return await modify("POST", route, values);
+};
+
+const put = async (route, values) => {
+  return await modify("PUT", route, values);
+};
+
+const postForm = async (route, values) => {
```

Comment:
This will only ever be used by the sign in process, because authentication requests must be form encoded instead of being in JSON.